packages/typespec-test/test/widget_dpg/generated/typespec-ts/review/widget_dpg.api.md

```diff
@@ -118,14 +103,15 @@ export interface WidgetsDeleteWidgetOptionalParams extends OperationOptions {
 
 // @public (undocumented)
 export class WidgetServiceClient {
-    constructor(endpoint: string, credential: KeyCredential, options?: WidgetServiceClientOptionalParams);
+    constructor(endpointParam: string, credential: KeyCredential, options?: WidgetServiceClientOptionalParams);
     readonly budgets: BudgetsOperations;
     readonly pipeline: Pipeline;
     readonly widgets: WidgetsOperations;
 }
 
 // @public
 export interface WidgetServiceClientOptionalParams extends ClientOptions {
+    apiVersion?: string;
```

Comment:
it seems we promot the apiVersion to client level, i was wondering if this is a case that not all operations have the apiVersion so we didn't do promotion before?